src/simmate/apps/warrenapp/workflows/relaxation/pbesol.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+
+from simmate.apps.warrenapp.workflows.relaxation.pbe import Relaxation__Warren__Pbe
+
+
+class Relaxation__Warren__Pbesol(Relaxation__Warren__Pbe):
```

Comment:
the naming guides I set up in the docs ([here](https://jacksund.github.io/simmate/full_guides/workflows/workflow_names/)), would mean you have to use `Vasp` as the "app" name. Following those rules, your workflow would be something like `Relaxation__Vasp__WarrenLabPbesol` --> `relaxation.vasp.warren-lab-pbesol`. 

But I am open to changing these rules. It does make sense to have something like `Relaxation__WarrenLab__Pbesol` -- I'm not sure we'd actually need to indicate `vasp` anywhere. Curious what your thoughts are...?

Reply:
I'm leaning towards the `Relaxation__Vasp__WarrenLabPbesol` convention. If we ever implemented a connection to another DFT package the naming helps keep it clear that VASP needs to be installed. I also think this makes it a little clearer that this workflow is just a VASP settings change from the matproj options. The main downside I see right now is that we lose some clarity on what app is being used. For example, `Relaxation__WarrenLab__Pbesol` implies that the workflow is coming from the warren_lab app while `Relaxation__Vasp__WarrenLabPbesol` implies to me that its coming from the vasp app. 

Maybe the solution is to remove the warren_lab app entirely and move most of its workflows into the vasp app. The badelf workflows and model could be moved to the badelf app. That keeps better with most of the apps being connections to 3rd party programs like bader, deepmd, etc. or being connections to much more complex workflows/scripts like in the evolution, clease, and badelf apps.

Reply:
makes sense. then let's stick with the `Relaxation__Vasp__WarrenLabPbesol` approach for now

> Maybe the solution is to remove the warren_lab app entirely and move most of its workflows into the vasp app

I would wait before moving these because there's a chance we do the opposite. For example I was considering moving all of the MatProj-based workflows into their own app (and out of the vasp app). Then we can clarify what is going on to the user in the doc website when a user registers the app.